apps/wallet-mobile/src/features/Send/useCases/ListAmountsToSend/ListAmountsToSendScreen.tsx

```diff
@@ -177,7 +177,7 @@ export const useSendTx = (
     refetchOnMount: false,
     refetchOnReconnect: false,
     queryKey: [wallet.id, 'send-tx'],
-    queryFn: () => wallet.createUnsignedTx(entry),
+    queryFn: () => wallet.createUnsignedTx([entry]),
```

Comment:
can you check if we can use the entries directly?

https://github.com/Emurgo/yoroi/blob/bfeb81ff93dca9324c19cb54e5ed7587913b13de/apps/wallet-mobile/src/features/Send/useCases/ListAmountsToSend/ListAmountsToSendScreen.tsx#L53